apps/vscode/command_client/command_client.py

```diff
@@ -357,6 +358,14 @@ def trigger_command_server_command_execution():
         was written to the file.  For internal use only"""
         actions.key("ctrl-shift-f17")
 
+    def emit_pre_phrase_signal():
+        """Touches a file to indicate that a phrase is about to begin execution"""
+        pass
```

Comment:
I suspect that `return None` would work here to make Talon think this method is implemented. Probably want a comment explaining that we're deliberately creating a no-op action, not an unimplemented action.

Reply:
I've added a comment to the global scope implementation.  I think I'd lean towards keeping that impl; otherwise it feels like we're trying to trick Talon's static analysis